src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -14,6 +14,10 @@ class PDOIntegration extends Integration
 
     const CONNECTION_TAGS_KEY = 'connection_tags';
 
+    const ENGINE_TO_SYSTEM_ARRAY = [
+        'mysql' => 'mysqldb',
+    ];
```

Comment:
As far as I understood it, "driver" is a better name than "engine". You can find the driver names in the docs:

https://www.php.net/manual/en/pdo.drivers.php

Then for each one, look for their `*_DSN` docs.

Reply:
This should be all set.